projects/plugins/jetpack/extensions/blocks/paywall/editor.js

```diff
@@ -1,8 +1,12 @@
 import { unregisterBlockType } from '@wordpress/blocks';
 import { subscribe, select } from '@wordpress/data';
 import { store as editorStore } from '@wordpress/editor';
-import registerJetpackBlock from '../../shared/register-jetpack-block';
-import { name, settings } from '.';
```

Comment:
👋 @monsieur-z, I noticed that `name` is being used in line 21:
https://github.com/Automattic/jetpack/blob/30f477439a16f9720b63523b64a314846e02483a/projects/plugins/jetpack/extensions/blocks/paywall/editor.js#L21

This produces a crash in the Jetpack app. Could we revert this change and keep the `name` property. Thanks 🙇 !

Reply:
I went ahead and created a PR to fix this: https://github.com/Automattic/jetpack/pull/34274.
Let me know if you could take a quick look, thanks 🙇  !